cr-examples/onnx/src/main/java/oracle/code/onnx/ir/OnnxOps.java

```diff
@@ -36,6 +36,140 @@ public final class OnnxOps {
 
     private OnnxOps() {}
 
+    @OpFactory.OpDeclaration(If.NAME)
```

Comment:
LIke with the operator API, this class should not be modified. We should use the same pattern.